docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+| exclamation point (!) | Avoid using exclamation points in general and never use more than one.
+| hyphenation (-)	| Avoid using hyphenated compound words. Use hyphens only if the meaning is unclear without them or if the only recognized form of the word includes a hyphen.
+| quotation marks (“ “)	| Avoid using quotation marks unless you need to quote a message or as string that would otherwise be confusing given its surrounding context.
+| semicolons (;) | Don't use semicolons instead of commas to separate items in a list. If you think the content should use semicolons, consider rewriting it into subtopics or an unordered bullet list.
+| slashes (/) and backslashes (\)	| Avoid using slashes or backslashes except when documenting paths that require either forward or backward slashes. Never use and/or in documentation.
+
+### Software versions
+
+Use *or later* or *and later* to refer to multiple versions of software. For example:
+
+- Firefox 3.6 or later
+- Rust compiler, version 1.55.0 and later
+
+### Tense
+
+Use present tense whenever possible. Use past tense only if you must describe something 
+that has already occurred.
+Use future tense only if you must describe something that has not yet occurred but can 
+be safely assumed.
+
+### User interface elements
+
+In general, you should avoid writing about user interface elements themselves. Instead, 
+documentation should always focus on what the audience needs to do or wants to 
+accomplish—that is, the user interaction—and not the widgets displayed on the screen.
+
+When referring to interactions in the user interface, you should generally follow the 
+capitalization that is used for the element unless it is rendered as all capital letters. 
+Because using all capitals is typically considered aggressive in text and most
+user interface elements that use all capital letters are also text that the user interacts
+with, you can break the "follow the UI rule" and use an initial capital letter.
+
+| Element	| How to use it
+| ------- | -------------
+| button	| Use bold for the button label. Don't include "button" in the description. For example: Click **Submit**.
+| checkbox	| Use checkbox, not box or check box, if you need to refer to a checkbox in the user interface. Use select and clear with checkboxes, not turn on and turn off, check and uncheck, or unselect and deselect.
+| click	| Use click to describe taking action on a standalone button. Do not use click on. Click and select are not interchangeable.
+| dialog	| If you need to refer to a dialog box, use dialog. Don't use pop-up window, dialog box, or dialogue box.
+| dropdown	| Use dropdown as an adjective, not as a noun. For example, use dropdown list.
+| menu | Use bold for the menu name and menu item names if the user interacts with them.
```

Comment:
As noted above, I think the mentioning of words or text examples without quotes or italics makes this harder to read, and reminds me of something out of recent literary fiction (e.g., Cormac McCarthy) rather than prose intended for clarity. Is this a technical writing best practice? If you have any other guidance on this practice from external sources that you can share, though, I'm willing to change my mind. 

This comment also applies to the two tables below.